src/main/java/javax/servlet/http/HttpServletMapping.java

```diff
@@ -97,13 +97,13 @@
 
     /**
      * <p>
-     * Return the portion of the URI path that caused this request to be matched. If the {@link #getMappingMatch} value
-     * is {@code
-     * CONTEXT_ROOT} or {@code DEFAULT}, this method must return the empty string. If the {@link #getMappingMatch} value
-     * is {@code
-     * EXACT}, this method must return the portion of the path that matched the servlet, omitting any leading slash. If
-     * the {@link #getMappingMatch} value is {@code EXTENSION} or {@code PATH}, this method must return the value that
-     * matched the '*'. See the class javadoc for examples.
+     * Return the portion of the URI path that caused this request to be matched. If the {@link #getMappingMatch} value is
+     * {@code
```

Comment:
This formatting still looks a bit weird, should there actually be a line break here?

Reply:
Ideally, there should not. That reformatting was the result of applying the standard EE4J formatting ruleset for the Eclipse IDE to the file. I noticed the extra line breaks but opted not to fix them. On one hand, it doesn't look quite right in the source code. On the other it won't affect the look of the Javadoc the user sees and manually 'fixing' the formatting after applying an automated tool (and having to fix them after every future reformatting) seemed a little pointless.